.github/workflows/devcontainer-smoke-test.yml

```diff
@@ -33,6 +33,10 @@ jobs:
       - name: Generate rails app sqlite3
         run: bundle exec railties/exe/rails new myapp_sqlite --database="sqlite3" --dev
 
+      - name: Remove old deprecated docker-compose
+        run: |
+          sudo rm /usr/local/bin/docker-compose
```

Comment:
removed the old and deprecated docker-compose 1.29.1 (from may 10, 2021), so it uses the compose v2 https://docs.docker.com/compose/migrate/